### PR TITLE
updating the seed. now cards have photos

### PR DESCRIPTION
**Pull request by paulazenanguyen**

### CODE REVIEW

**Review by WalkerTexas-coder — DISMISSED** [review]

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -21,19 +21,44 @@ body {
   font-size: 17px;
   color: gold;
 }
-
+// .footerbackground{
+//   background-image: url(https://images.unsplash.com/photo-1516731566880-919c39006c9d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80);
+//   background-size: 100% 100%;
+//   background-attachment: fixed;
+// }
```
Comment (sjproctor) [inline app/assets/stylesheets/application.scss]: Commented out code should be removed from the PR.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -21,19 +21,44 @@ body {
...
 .welcome {
   display: inline-block;
   width: 50%;
+  color: black;
+}
+.headerbackground{
+  background-image: url(https://images.unsplash.com/photo-1516118461620-03680a03bd27?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2946&q=80);
+  background-size: 100% 100%;
+  // background-repeat: no-repeat;
+  // background-attachment: fixed;
```
Comment (sjproctor) [inline app/assets/stylesheets/application.scss]: Same thing here. Commented out code should not be pushed to main.

File: app/javascript/components/components/Header.js [hunk 3]
```diff
@@ -12,49 +12,51 @@ class Header extends Component {
...
 
+      <div className="headerbackground">
       <div className="header" id = 'full-page'>
         <h1 className="welcome">TEAMLY</h1>
         <h2 className="welcome" align="right">Hello {logged_in ? this.props.current_user.email : 'guest'}!</h2>
         <img src= "https://images.unsplash.com/photo-1569517282132-25d22f4573e6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1866&q=80" height="100px" width="100px"/>
         <Nav>
           <NavItem>
-            <NavLink href="/">Home</NavLink>
+            <NavLink href="/" id='nav-text'>Home</NavLink>
```
Comment (sjproctor) [inline app/javascript/components/components/Header.js]: You should be consistent with your quotations. Looks like you are using double quotes for most of your tags.

**Review by sjproctor — CHANGES_REQUESTED** [review]
Need to removed some comments!

**Review by sjproctor — APPROVED** [review]

**Pull request merged**